guides/common/modules/con_requirements-for-installation-in-an-ipv6-network.adoc

```diff
@@ -21,5 +21,5 @@ ifndef::satellite[]
 Note that this requirement is for Katello users only.
```

Comment:
Unrelated, but we should improve conditionals here so we only show the requirement if needed. Then drop this line.

I think line 14 can be changed to `ifndef::satellite`. While it mentions content, it is done in a way that is generic enough to also apply to non-Katello.

Reply:
Addressed :+1: